packages/itwinui-react/package.json

```diff
@@ -80,10 +80,8 @@
     "@floating-ui/react": "^0.26.3",
     "@itwin/itwinui-illustrations-react": "^2.1.0",
     "classnames": "^2.3.2",
-    "jsdom": "^24.0.0",
     "react-table": "^7.8.0",
-    "react-transition-group": "^4.4.5",
-    "tslib": "^2.6.0"
```

Comment:
Could you undo this? `tslib` is intentionally made part of deps. It helps make the compiled `tsc` output smaller

Reply:
Oh, okay, got it, I searched for import references, there were none, I thought it was an accident.
Reverted